### DISCUSSION TITLE
Docker image

### DISCUSSION THREAD

**Discussion by rtm516** [pelican-dev/panel#226] [General]
Is there a technical reason the Docker Image isn't built or published for the panel anywhere? If not would you be open to a PR adding the build workflow for it?

**Comment by rtm516** [comment]
It was removed by this commit but doesnt look like there was a reason https://github.com/pelican-dev/panel/commit/3a951059310817900c0427faa149ee030f2e8a9b

**Comment by Boy132** [comment]
> Is there a technical reason the Docker Image isn't built or published for the panel anywhere? [...]

Yes, it needs some refactors. For now the focus was on the non-dockerized install method.
Later the dockerized install will be the preferred method of install.
